src/Idris/REPL/Commands.hs

```diff
@@ -17,6 +17,7 @@ data Command = Quit
              | Reload
              | Watch
              | Load FilePath (Maybe Int) -- up to maximum line number
+             | RunShellCommand FilePath
```

Comment:
Conceptually, I suppose it could be split into `(String, [String])`. I opted not to do that, since:

1. I'm ultimately passing the whole thing to [`system`](http://hackage.haskell.org/package/process-1.4.3.0/docs/System-Process.html#v:system), which takes a single `String` as an argument (this is the [same way that GHCi does it](http://git.haskell.org/ghc.git/blob/9fd87ef8a16fbbce35205ae63d75d239bb575ccc:/ghc/GHCi/UI.hs#l1204)).
2. It integrates nicely with `strArg`, which takes one `String` argument.

So splitting it up into `(String, [String])` seems silly, given that its only purpose to be the payload for `system`.